### PR TITLE
Views

### PR DESCRIPTION
**Pull request by SelmaNdapanda**
### In this pull request, I implemented the following:  :arrow_down: 

- Inserted User, posts, and comments data in the database
- Created view files and style them
- Updated the models and controllers to ensure methods work on the view

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,33 @@
...
+      <div class="user-likes">Likes: <%= @user.likes_counter %></div>
+    </div>
+    <div class="user-content">
+      <div class="user-text"><%= @user.text %></div>
+      <div class="user-date">Date: <%= @user.created_at %></div>
+    </div>
+
+    <div class="user-posts">
+      <div class="posts-container">
+        <% @user.posts.each do |post| %>
```
Comment (Davidosky007) [inline app/views/users/show.html.erb]: * As shown in the wireframe you should render the 3 most recent posts as you have created the method already in your user model also ensure your views function as shown in the wireframe below 👇 
image

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1 +1,30 @@
...
+        <div class="post-likes">Likes: <%= @post.likes_counter %></div>
+        </div>
+        <div class="post-content">
+        <div class="post-text"><%= @post.text %></div>
+        <div class="post-date">Date: <%= @post.created_at %></div>
+        </div>
+    </div>
+    <div class="post-comments">
+        <div class="comments-container">
+        <% @post.comments.each do |comment| %>
```
Comment (Davidosky007) [inline app/views/posts/show.html.erb]: * As shown in the wireframe below you are to render 5 most recent comments as you have created the method already in your post model, Kindly ensure your views function as shown in the wireframe below 👇 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @SelmaNdapanda 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking readme file 👍 

## Required Changes ♻️
* Ensure your views functions as shown in the wireframe as currently only the view for all users is working as expected 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @SelmaNdapanda, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**